api/lib/routes/routeHelper.js

```diff
@@ -31,3 +31,20 @@ get any error during the schedule creation/update. */
       callback(error,{ 'statusCode':HttpStatus.INTERNAL_SERVER_ERROR });
     });
 }
+
+exports.deletePolicy = function(req, schedulerResponse, callback) {
```

Comment:
I will create a new CL for the PUT changes for this unused object. The delete API is done.